app/config/findbugs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+FindBugs exclude filter
+Severity: HARD
+-->
+<FindBugsFilter>
```

Comment:
data binding classes are auto generated and are present always. please take a look at this https://github.com/andriydruk/RxDNSSD/blob/master/library/findbugs-filter.xml  @iamareebjamal